content/docs/2.14/scalers/dynatrace.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Dynatrace"
+availability = "2.14+"
+maintainer = "Community"
+description = "Scale applications based on Dynatrace metric data points"
+go_file = "dynatrace_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `dynatrace` trigger that scales based on Dynatrace metric data points.
+
+```yaml
+triggers:
+  - type: dynatrace
+    metadata:
+      host: https://dummy-instance.live.dynatrace.com/
```

Comment:
Is this host always using `live.dynatrace.com` or not? Otherwise, we should just use this:
```suggestion
      organization: dummy-instance
```

Reply:
It depends on the [deployment type](https://docs.dynatrace.com/docs/get-started/monitoring-environment#melocation), so not necessarily

Reply:
In that case, why not introduce `deploymentType` or so which is cloud by default and uses `organization`. If it is not, then host is rquired to be filled in?

It feels like a nicer UX if `organization` is used by default.

Reply:
The `organization` terminology isn't really used in Dynatrace afaik, so I'm not so sure about it. Plus it would require to configure 2 parameters instead of 1, if I understood correctly. I tried to follow a similar logic to other scalers, which usually have a `host` parameter (admittedly the naming is anything but consistent across scalers).

Reply:
Fair enough, but it would have made the usability easier though